### DISCUSSION TITLE
Key not present in table

### DISCUSSION THREAD

**Discussion by RodPaDev** [sequelize/sequelize#14673] [Help & Questions] [answered]
I'm trying to sync my DB and seed it after.

I have two tables `Sales` & `Product` and they have a one-to-many relationship.

which is defined like so (according to the docs)
```js
Product.hasMany(Sales)
Sales.belongsTo(Product)
```

This seems to work fine as the `Sales` table has a `productId` after sync.

The problem comes when I try to seed the DB (local development).
```
Key (productId)=(TI_WsmX-K7DfrNH4m2jzI) is not present in table "product".
```

To me, the error says that no row in the product has a PK matching productId, so I tried to disable foreign-key checks by doing this
```SQL
SET FOREIGN_KEY_CHECKS = 0
```
but I get this error instead
```
error: unrecognized configuration parameter "foreign_key_checks"
```

Any ideas on how I could get past this?

**Comment by ephys** [comment, marked as answer]
Which database are you using? Isn't FOREIGN_KEY_CHECKS a sqlite config option?
I wouldn't recommending disabling them though, you should create the product entry that is being referenced first
